src/components/TextField.vue

```diff
@@ -400,6 +410,12 @@ export default {
     }
   }
 
+  &--required {
+    .text-field__label span {
```

Comment:
```suggestion
    .text-field__required-text {
```

Reply:
FYI the specificity of what you've got is 3, one from the text-field--required, one from `text-field__label` and another from `span`.

BEM says you should always have specificity of 1, unless there's a modifier involved, where it can be 2. `&--required` is the modifier here.